### DISCUSSION TITLE
Init SceneScope without Boot/Loader scene

### DISCUSSION THREAD

**Discussion by jjpb44** [gustavopsantos/Reflex#30] [Q&A]
I've followed the example in readme and i'm wondering if its possible to init a SceneScope container without needing to start in a Boot scene with loader managing the start of a scene?

I tried just adding a SceneScope with simple bindings installer in sample project (without project scope or boot) and the container is not initialized.

**Comment by gustavopsantos** [comment]
What exactly do you mean by container being initialized? To actually register the components and to create the specific container?
If yes, then yes, thats possible, you just need to create a MonoBehaviour implementing IInstaller. Add your bindings there. And remember to add this component as a child to your SceneScope.
@jujpenabe  Try this and let me know ;)

**Reply by jjpb44** [reply]
It's strange, I tested the exact same steps on another project and it works as expected, On the other project its impossible to Install the SceneScope Bindings directly from the scene.

**Comment by gustavopsantos** [comment]
Just a quick tip, you can join our discord server, its faster for quick questions like this one ;)
https://discord.gg/XM47TsGScH

**Reply by jjpb44** [reply]
Thanks!, I'll join.

**Reply by gustavopsantos** [reply]
Ping me there directly, I wanted to take a look on this issue more closely if thats possible.
